packages/fetchai/connections/gym/connection.py

```diff
@@ -29,7 +29,9 @@
 
 from aea.configurations.base import ConnectionConfig, PublicId
 from aea.connections.base import Connection
+from aea.crypto.wallet import CryptoStore
 from aea.helpers.base import locate
+from aea.identity.base import Identity
 from aea.mail.base import Address, Envelope
 
 from packages.fetchai.protocols.gym.message import GymMessage
```

Comment:
Note, as you're bumping the version, please provide a `PUBLIC_ID` constant here (see other connections); which is now the recommended pattern.